app/code/Magento/Catalog/Model/Product/Option/Type/Date.php

```diff
@@ -340,7 +340,7 @@ public function is24hTimeFormat()
      */
     public function getYearStart()
     {
-        $_range = explode(',', $this->getConfigData('year_range'));
+        $_range = explode(',', $this->getConfigData('year_range') ?? '');
```

Comment:
Hello @SilinMykola 
What do you think about suggestion below?

```suggestion
        $_range = $this->getConfigData('year_range') !== null ? explode(',', $this->getConfigData('year_range')) : [];
```

Reply:
Hello @andrewbess 
Thanks for your review. I made changes, please check.